### ISSUE TITLE
Support for WebViewRuntime2 / blazor hybrid

### ISSUE THREAD

**Issue by ysmoradi** [velopack/velopack#6]
Please add support for [WebViewRuntime2](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution)
```
dotnet vpk pack --framework EdgeWebViewRuntime2
```

**Comment by caesay** [comment]
Released in [0.0.208-gd4b199d](https://www.nuget.org/packages/Velopack/0.0.208-gd4b199d)

The argument is `--framework webview2`

**Comment by ysmoradi** [comment]
I'm running the following command:
```bash
dotnet vpk pack -u MyApp -v 1.0.0 -p .\publish -e MyApp.exe -r win-x86
```
Having `-r win-x86` proves that I'm using the latest pre-release and x86 setup installer works well, but If I want to pass `--framework webview2` I'll receive the following error:

```
Invalid target dependency string: webview2.
```

**Comment by caesay** [comment]
Fixed in 0.0.212
